src/lexicon/app/bsky/feed.rs

```diff
@@ -172,3 +173,129 @@ pub struct GetPostThread {
 pub struct GetPostThreadOutput {
     pub thread: ThreadViewPostEnum,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Feed {
+    /// format: at-uri
+    pub uri: String,
+}
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Links {
+    #[serde(rename = "privacyPolicy")]
+    pub privacy_policy: Option<String>,
+    #[serde(rename = "termsOfService")]
+    pub terms_of_service: Option<String>,
+}
+
+/// app.bsky.feed.describeFeedGenerator
+/// Returns information about a given feed generator including TOS & offered feed URIs.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DescribeFeedGeneratorOutput {
```

Comment:
Note that `describeFeedGenerator` has no `input`. Should that change naming at all? Probably not, but wanted to flag.